src/models/AppState.js

```diff
@@ -422,6 +431,69 @@ function (Okta, Q, Factor, BrowserFeatures, Errors) {
           return userProfile.firstName + ' ' + userProfile.lastName;
         }
       },
+      'expiresAt': {
+        deps: ['lastAuthResponse'],
+        fn: function (res) {
+          if (!res.expiresAt) {
+            return false;
+          }
+          return res.expiresAt;
+        }
+      },
+      'target': {
+        deps: ['lastAuthResponse'],
+        fn: function (res) {
+          if (!res._embedded || !res._embedded.target) {
+            return false;
```

Comment:
prefer to match the type. looks like `res._embedded.target` is an object but here returns boolean.
and several other places below.

Reply:
good point! will update with null